modules/migration-about-migrating-applications-api.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration-toolkit-for-containers/advanced-migration-options-mtc.adoc
+
+[id="migration-about-migrating-applications-api_{context}"]
+= About migrating applications
+
+You can migrate applications from a local cluster to a remote cluster, from a remote cluster to a local cluster, and between remote clusters.
+
+[id="terminology_{context}"]
+== Terminology
+
+Source cluster::
+* Cluster from which the applications are migrated.
+
+Destination cluster::
+* Cluster to which the applications are migrated.
+
+Replication repository::
+* Object storage.
+* Requires network access to all clusters.
+
+Indirect migration::
+* Images, volumes, and Kubernetes objects are copied from the source cluster to the replication repository and then from the replication repository to the destination cluster.
+
+Direct volume migration::
+* Volumes are copied directly from the source cluster to the destination cluster.
+* Significantly faster than indirect migration.
+
+Direct image migration::
+* Images are copied directly from the source cluster to the destination cluster.
+* Significantly faster than indirect migration.
+
+Host cluster::
+* Cluster on which the `migration-controller` pod and the web console run.
+* Usually the same as the destination cluster and the local cluster but this is not a requirement.
+* Does not require an exposed secure registry route for direct image migration.
+
+Local cluster::
+* Usually the same as the destination cluster and the host cluster but this is not a requirement.
```

Comment:
I think the "local cluster" section should be removed. I don't think we have a separate concept of "local cluster" that is distinct from "remote cluster". The cluster that is not a remote cluster is the host cluster.